src/backend/utils.ts

```diff
@@ -1288,6 +1290,69 @@ async function getPathDiskSize(path: string): Promise<number> {
   return statData.size
 }
 
+export async function checkRosettaInstall() {
+  if (!isMac) {
```

Comment:
this is only called inside an `if (isMac)` check, so we don't need to check again (or we don't need the other check)